src/main/resources/com/google/api/codegen/java/disco_schema.snip

```diff
@@ -0,0 +1,141 @@
+@extends "java/common.snip"
+@extends "java/method_sample.snip"
+
+@snippet generate(apiFile)
+  {@renderFileHeader(apiFile.fileHeader)}
+
+  @@Generated("by GAPIC")
+  @@BetaApi
+  public final class {@apiFile.schema.typeName} implements Serializable {
+    {@members(apiFile.schema)}
+    {@constructor(apiFile.schema)}
+    {@getters(apiFile.schema)}
+    {@newBuilder(apiFile.schema)}
+    {@builder(apiFile.schema)}
+    {@toString(apiFile.schema)}
+    {@equalsMethod(apiFile.schema)}
+    {@hashCode(apiFile.schema)}
+  }
+@end
+
+@private toString(xschema)
```

Comment:
The "x" prefix thing was something I started a while back but decided to stop using, and I haven't gotten a chance to remove it everywhere yet. Go ahead and remove it from your code (so use `schema` here).